tests/atest/rules/errors/invalid_setting_in_resource/test_rule.py

```diff
@@ -0,0 +1,9 @@
+from tests.atest.utils import RuleAcceptance
+
+
+class TestRuleAcceptance(RuleAcceptance):
+    def test_rule(self):
+        self.check_rule(expected_file="expected_output_rf6.txt", target_version="==6.*")
```

Comment:
what about RF 7? Maybe it should be:

```suggestion
        self.check_rule(expected_file="expected_output_rf6.txt", target_version=">=6")
```